gcp_variant_transforms/vcf_to_bq.py

```diff
@@ -224,7 +224,8 @@ def run(argv=None):
       known_args.use_allele_num,
       known_args.minimal_vep_alt_matching,
       known_args.infer_annotation_types,
-      counter_factory)
+      counter_factory,
```

Comment:
It may not related with this PR, but is there any reason why not create the counter inside `ProcessedVariantFactory`?

Reply:
Good question. The main reason is to share the factory across all classes (currently only one classes uses counters, but we have some open issues to add counters to other classes too) and also that we want to override it in tests to use the `NoOpCounterFactory` (or any other custom metric factory).

Reply:
Good point! Thanks!